.github/workflows/config.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
You can remove all the comments. It feels that the commands are quite self explanatory.